### DISCUSSION TITLE
김동아  1회차 블로그 피드백 요청합니다

### DISCUSSION THREAD

**Discussion by ehddk** [Orm-camp/blog-feedback#11] [동글동글 2기 피드백]
[회고록](https://tofu15.tistory.com/10)
미니 프로젝트 회고록을 키워드 중심으로 간략하게 기재해봤습니다!
분명 작성할때는 공백,띄어쓰기가 없이 잘 나오는데 글을 등록하고나면 공백이 막 생기네요 ㅠㅠ

**Comment by taeyoung0001** [comment]
동아님 글 잘 읽었습니다.
글도 체계적으로 잘 잡혀있는거 같습니다. 아쉬운 점과 좋았던 점 내용에 하이라이트를 줘서
눈에 확 들어오게 하신 부분 배워갑니다.. 내용도 너무 좋네요

동아 팀장님의 좋은 리더쉽으로 인해 더욱 더 좋은 결과물이 나왔던거 같네요  :ㅇ
디스코드에서도 저런 이모티콘 자주 사용하시는거같은데 블로그에서도 나왔네요 ㅋㅋㅋ
앞으로 더욱 화이팅 해봅시다!
